lib/screens/explore_screens/unplant_seeds/components/claim_seeds_succes_dialog.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:seeds/components/custom_dialog.dart';
+import 'package:seeds/datasource/local/models/fiat_data_model.dart';
+import 'package:seeds/datasource/local/models/token_data_model.dart';
+import 'package:seeds/i18n/explore_screens/plant_seeds/plant_seeds.i18n.dart';
+import 'package:seeds/navigation/navigation_service.dart';
+
+class ClaimSeedsSuccessDialog extends StatelessWidget {
+  final TokenDataModel claimSeedsAmount;
+  final FiatDataModel claimSeedsAmountFiat;
+
+  const ClaimSeedsSuccessDialog({Key? key, required this.claimSeedsAmountFiat, required this.claimSeedsAmount})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return WillPopScope(
+      onWillPop: () async {
+        Navigator.of(context).pop();
```

Comment:
should this not be the same as 

onSingleLargeButtonPressed: () {
          Navigator.popUntil(context, ModalRoute.withName('app'));
          NavigationService.of(context).navigateTo(Routes.unPlantSeeds);
          },

Reply:
Yes it should be the same. Fixed, 👍